src/pageHead.test.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Link } from 'react-router';
+import TestUtils from 'react-addons-test-utils';
+import { findComponentsWithType } from 'meetup-web-mocks/lib/testUtils';
+import PageHead, {
+	PAGE_TITLE_CLASS,
+	PAGE_ACTIONS_CLASS,
+} from './PageHead';
+import PageActionButton from './PageActionButton';
+import Popover from './Popover';
+
+const menu = [
+		<PageActionButton icon='search' label='Search' />,
+		<PageActionButton icon='edit' label='Edit' />,
+		<Popover
+			align='left'
+			trigger={
+				<PageActionButton icon='overflow' label='More' />
+			}
+			menuItems={[
+				<Link to='somepath/'>First option</Link>,
+				<Link to='somepath/'>Second option</Link>,
+				<Link to='somepath/'>Third option</Link>,
+			]}
+		/>
+	],
+	pageTitle = 'Page title',
+	subtitle = 'Sub title';
+
+let pageHead,
+	pageHeadEl;
+
+describe('PageHead', function() {
+	beforeEach(() => {
+		pageHead = TestUtils.renderIntoDocument(
+			<PageHead title={pageTitle} />
+		);
+		pageHeadEl = ReactDOM.findDOMNode(pageHead);
+	});
+
+	afterEach(() => {
+		pageHead = null;
+		pageHeadEl = null;
+	});
+
+	it('exists', function() {
+		expect(pageHeadEl).not.toBeNull();
+	});
+
+	describe('title', () => {
+		it('should display a title area', () => {
+			expect(pageHeadEl.innerHTML.indexOf(pageTitle)).not.toBe(-1);
+		});
+		it(`should have a 'Flex' component with a class of ${PAGE_TITLE_CLASS}`, () => {
+			const flexItems = findComponentsWithType(pageHead, 'Flex');
+			const pageTitle = flexItems.filter(flex => flex.props.className.indexOf(PAGE_TITLE_CLASS) !== -1);
+			expect(pageTitle.length).toBe(1);
+		});
+		it(`should NOT have a 'Flex' component with a class of ${PAGE_ACTIONS_CLASS}`, () => {
+			const flexItems = findComponentsWithType(pageHead, 'Flex');
+			const pageActions = flexItems.filter(flex => flex.props.className.indexOf(PAGE_ACTIONS_CLASS) !== -1);
+			expect(pageActions.length).toBe(0);
+		});
+		it('should NOT have a \'<p>\' tag', () => {
```

Comment:
why can't this PageHead ever contain a `<p>`? Is this intended to ensure that the `subtitle` is not rendered? If so, give the subtitle a classname and check for the non-existence of that